mappings/net/minecraft/util/registry/SimpleRegistry.mapping

```diff
@@ -46,3 +50,8 @@ CLASS net/minecraft/class_2370 net/minecraft/util/registry/SimpleRegistry
 		ARG 1 key
 	METHOD method_40277 (Lnet/minecraft/class_5321;)Lnet/minecraft/class_6880$class_6883;
 		ARG 1 key
+	METHOD method_40561 getEntriesList ()Ljava/util/List;
```

Comment:
Q. Should we rename this to `getEntries` and rename existing `getEntries` to `getEntrySet`?

Reply:
```suggestion
	METHOD method_40561 getEntries ()Ljava/util/List;
```